wagtail/contrib/postgres_search/query.py

```diff
@@ -0,0 +1,124 @@
+# From https://github.com/django/django/pull/8313
```

Comment:
Should we include a reminder here that this can be removed when/if the Django PR is accepted?